### DISCUSSION TITLE
Keyball61_OLED付近のジャンパ部分_パッド剥離時の対処法をご教示いただけますでしょうか

### DISCUSSION THREAD

**Discussion by JinBallMen** [Yowkees/keyball#277] [Q&A]
お世話になっております。トラックボールなし側の基盤で、以下1枚目のパットが剥離してしまい、問い合わせさせていただきました。

すみません、もともとこちらに投稿したときは#258や#125をベースに質問をしていましたが、おそらく位置が違うだろうと思い
さらに調べて問い合わせ内容を変えました。

調べた事としては、ガーバーデータ？が見れるソフトから確認しました。

見たままかと思いますが、実際にトラックボールなし側と比較すると以下だと考えています。
以下のようにつなげばよろしいでしょうか。

![20230718_013412~2](https://github.com/Yowkees/keyball/assets/75989686/89cec622-e435-4744-99b6-1737fba2b12a)

![kairozu](https://github.com/Yowkees/keyball/assets/75989686/c2baed40-e1c2-4a99-b07d-1203056437a3)
![tunagari](https://github.com/Yowkees/keyball/assets/75989686/33dea500-72b8-4c1d-8b66-4c43f92d7b9f)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

おっとっと、違いますね。
剥離している一番左のジャンパーはGNDになりますので、下記画像の２点をエナメル線等で接続していただければ、ジャンパーしたのと同じになります。
なお、この作業はタクトスイッチや4Pピンソケットを半田付けしてから行ってください。

以上、お手数ですがご対応お願いいたします。
![kb61_jampfix](https://github.com/Yowkees/keyball/assets/84209772/af915ba5-f83b-4e1b-b2fa-205fef36d1bd)

**Reply by JinBallMen** [reply]
回答頂きありがとうございます。対応いたします。
また、大変お恥ずかしながら以下も剥離してしまいました。どちらもOLED付近です。

・一枚目　左トラックボールなしの右端
・二枚目　右トラックボールありの右端


一枚目は#218の問い合わせと同様の対応、2枚目は回答頂いた箇所と同様、RESET部分の下側に繋ぎますでしょうか。
また、初心者質問ですみませんが
先にはんだを入れてしまうと部品が入らなくなってしまう（＝そもそも対応する部品に線を繋げられなくなるので意味がない）ため、部品を取り付けた後に線を繋げるという理解で良いでしょうか。

![20230719_214853~2](https://github.com/Yowkees/keyball/assets/75989686/80388cbe-b013-4ab5-b2f5-6ea595a29fc6)

![20230719_215000~2](https://github.com/Yowkees/keyball/assets/75989686/43f12d0c-1fa9-47d9-a58d-19f2c8504b2f)

**Comment by Yowkees** [comment]
＞先にはんだを入れてしまうと部品が入らなくなってしまう（＝そもそも
＞対応する部品に線を繋げられなくなるので意味がない）ため、
＞部品を取り付けた後に線を繋げるという理解で良いでしょうか。
はい。すべての半田付け部品を取り付けた後に線を追加してください。

まず右手側は下記画像の緑丸と緑四角をエナメル線等でショートしてください。緑四角は２点ありますので、半田付けしやすほうを選んでください。
![KB61R_jampfix](https://github.com/Yowkees/keyball/assets/84209772/f0b8ec20-8cf1-427c-bc87-054b3530e395)

**Comment by Yowkees** [comment]
次に左手側は下記画像の緑点をエナメル線等でショートしてください。
以上、お手数ですがご対応お願いいたします。
![KB61L_jampfix](https://github.com/Yowkees/keyball/assets/84209772/b6d665a1-4fe2-4d4f-a3c8-5d7217cfbb73)

**Comment by JinBallMen** [comment]
ありがとうございました。無事つなげることができました。
